proto/configpb.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
+package configpb;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+service Config {
+    rpc Create(CreateRequest) returns (CreateResponse) {}
+    rpc Get(GetRequest) returns (GetResponse) {}
+    rpc Update(UpdateRequest) returns (UpdateResponse) {}
+    rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+}
+
+message Status {
+    enum StatusCode {
+        UNKNOWN = 0;
+        OK = 1;
+        STALE_VERSION = 2;
```

Comment:
We should change `STALE_VERSION ` to `WRONG_VERSION`, so we can handle both version too high or too low.
BTW should we also add a `version` field on DeleteResponse?